### PR TITLE
Autocomplete drop down now behaves as you would expect

### PR DESCRIPTION
**Pull request by trg**
Also it uses fetch now

### CODE REVIEW

**Comment by dankeezer** [comment]
Thanks for this! `fetch` is what I was looking for... 

I pulled this down and spun it up with `bin/dev`

Is this the behavior you're seeing? I feel like I'm missing something obvious. The links don't click though, which is same issue I had. 

Firefox (No [x] here either):
autocomplete

Safari (has the [x] but results never come back):
autocomplete-safari

Chrome, similar behavior to Safari.
autocomplete-chrome

**Comment by trg** [comment]
Good catches! I think my latest push should fix those bugs.

I think this dropdown is getting complex enough (what with the state management bugs) that it might be worth investing the time figuring out how to get React working in Rails 7.

File: app/views/search/index.html.erb [hunk 1]
```diff
@@ -1,39 +1,41 @@
 <div class="relative">
   <%= form_tag search_path, method: "get" do %>
-    <%= search_field_tag :query, nil, autocomplete: "off", placeholder: "Search Machines, Tournaments ...", class: "form-control w-full", onkeyup: "onKeyUp(this.value)" %>
+    <%= search_field_tag :query, nil, autocomplete: "off", 
+                                     placeholder: "Search Machines, Tournaments ...", 
+                                     class: "form-control w-full", 
+                                     onkeyup: "onKeyUp(this.value)",
+                                     onblur: "setTimeout(hideDropdown, 300);", # heh heh heh
```
Comment (dankeezer) [inline app/views/search/index.html.erb:7]: I think I read about this hack, clever!

**Review by dankeezer — APPROVED** [review]
Yes, I will be working on the react stuff shortly. I've been getting tests/linting/code coverage all setup in `dk/tests`. Unnecessary but a good exercise with all the new versions of all the things I used to use.

**Pull request merged**